lib/metaschema-config/config.js

```diff
@@ -84,6 +87,46 @@ const preprocessor = schemas => {
   return arr;
 };
 
+const validate = (schema, path) => {
+  const errors = [];
+
+  for (const [prop, value] of Object.entries(schema)) {
+    if (!value) continue;
+
+    const valueType = typeof value;
+    if (valueType === 'object') {
+      errors.push(...validate(value, `${path}.${prop}`));
+    } else if (valueType === 'function') {
+      const stringifyValue = value.toString();
+      let err;
+      try {
+        new Function(`(${stringifyValue})`);
+      } catch (e) {
+        err = e;
+      }
+
+      if (err && !value.prototype) {
```

Comment:
Could you please add comments to note exactly why there are two `try-catch` and why this `value.prototype` is relevant?